drafts/2019-11-11-tekton-architecture.adoc

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "Cloud-Native Devops: Tekton Architecture and Design"
+categories: blog
+author_picture: https://avatars2.githubusercontent.com/u/4930573
+author_github: https://github.com/mnuttall
+seo-title: Title - Kabanero.io
+seo-description: It's all about the architecture.
+blog_description: "It's all about the architecture"
+---
+= Tekton Architecture
+Mark Nuttall <https://github.com/mnuttall>
+
+== Introduction
+This blog post is part of a series about Tekton, the cloud-native CI/CD engine that underpins Kabanero. It leads on from link:/publish/2019-11-05-Tekton-Enterprise-CI-CD.adoc[our previous article], ‘How Tekton can power your Enterprise’s continuous integration & continuous delivery.’ We go into more detail on how Tekton works, look at how it can be integrated into an end-to-end DevOps process and finally look forward to the following articles that will flesh out its use in an enterprise context. 
```

Comment:
Suggest changing URL to point to https://kabanero.io/blog/2019/11/05/Tekton-Enterprise-CI-CD.html on the published kabanero website/blogs page